man/systemd.netdev.xml

```diff
@@ -1268,6 +1268,27 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AdActorSysPrio=</varname></term>
+        <listitem>
+          <para>Specifies the 802.3ad system priority. Ranges [1-65535].</para>
```

Comment:
Maybe `[0-65535]`?. At least, config parser does not warn when `0` is set.

Reply:
Then the kernel will complain. I guess I have to add a specific parser for this as well.

Reply:
Yeah, but, if `0` is set, then `networkd` does not send the value. So, how about mentioning `0` does not set priority?

Reply:
Ah, you have added a parser for the option. OK. (I think it may be overkill though...)